rules/files-not-contents-config.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Could this merge with file-not-contents? I know it calls that for functionality, but wondering if it can be integrated into the same API.

Reply:
Merged and updated PR.